data/doodlebug.json

```diff
@@ -0,0 +1,136 @@
+{
+  "name": "Doodlebug",
+  "tagline": "Hoppy Sub-Session Pale Ale.",
+  "first_brewed": "09/2015",
+  "description": "Doodlebug is our fusion of low strength and enormously high tropical hop levels – think a mashup of Hop Fiction and How to Disappear Completely, and you’re almost there. Clocking in at 2.8% ABV, Doodlebug is keg-only and is packed with Amarillo, Chinook and Mosaic; letting fly a hoppy buzzbomb at sub-session-strength.",
+  "abv": 2.5,
+  "ibu": 35,
+  "target_fg": 1008,
+  "target_og": 1027,
+  "ebc": 10,
+  "srm": 5,
+  "ph": 4.2,
+  "attenuation_level": 70.4,
+  "volume": {
+    "value": 20,
+    "unit": "liters"
+  },
+  "boil_volume": {
+    "value": 25,
+    "unit": "liters"
+  },
+  "method": {
+    "mash_temp": {
```

Comment:
This now needs to have an array wrapped around it, due to some later recipes having multiple mash steps check out the template.json